### ISSUE TITLE
2.9" BW display (HwType 33)

### ISSUE THREAD

**Issue by wizz666** [wiggertdehaan/open_epaper_link_homey#39]
Is it possible to add these displays to the app?
![BW displays](https://github.com/wiggertdehaan/open_epaper_link_homey/assets/23716728/58d2073c-58e5-43ec-8236-d2357c74f33f)

**Comment by wizz666** [comment]
An image of the back of the display if that helps. 
![BW lable](https://github.com/wiggertdehaan/open_epaper_link_homey/assets/23716728/a33b3592-0a8b-4aff-8b9c-0cafd7c6b405)

**Comment by wiggertdehaan** [comment]
https://github.com/jjwbruijn/OpenEPaperLink/blob/master/ESP32_AP-Flasher/data/tagtypes/21.json

**Comment by JohPePe** [comment]
Hi!
I've got 5 of them exactly like it. Unable to add them to Homey and OpenEPaperLink app.
If it could be fixed it would be great!

**Comment by JonteBerra** [comment]
Would be very nice if this typ would be compatible with homey! It´s the ZBS243-based tags like @wizz666 photo describe. Do you work with implement more tags in the future? Thanks for a great app!

**Comment by lolpold** [comment]
Agreed, I have this display and would love to get it to work in Homey.

**Comment by Blom1974** [comment]
Adding my wish for this as well.

**Comment by joamlin** [comment]
+1 on this, would be so nice to have this screen added - thanks!

**Comment by higgins2k** [comment]
Got a few of these as well, keeping my fingers crossed they can "talk" with homey in a near future. :)
I have an Yellow AP as well, I hope it'll work just fine with the homey.
Thanks in advance!

**Comment by wiggertdehaan** [comment]
Should be solved on `main` by the generic driver from #56.

Rather than adding a driver per model, there is now a single "OpenEPaperLink Tag" driver whose pairing list shows every tag the AP reports, whatever its hardware type. The type is read from the AP and stored on the device, so a display the AP can drive can be added to Homey without the app needing to know about that model in advance.

The six per-model drivers are marked deprecated: already-paired devices keep working and keep their flows, they are just no longer offered in Add Device.

Please give it a try after updating. If your BW 2.9" shows up in the AP but not in the pairing list, reopen and include what `/get_db` reports for it.
